ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -190,6 +261,43 @@ pub(crate) struct Instruction {
     name: RefCell<Option<String>>,
 }
 
+impl PartialEq for Instruction {
+    fn eq(&self, other: &Self) -> bool {
+        std::ptr::eq(self, other)
+    }
+}
+
+impl Instruction {
+    pub(crate) fn get_operand(&self, index: usize) -> &Operand {
+        self.operands.get(index).unwrap()
+    }
+
+    pub(crate) fn opcode(&self) -> Opcode {
+        self.opcode
+    }
+
+    pub(crate) fn is_store(&self) -> bool {
+        self.opcode == Opcode::Store
+    }
+
+    pub(crate) fn is_gep(&self) -> bool {
+        self.opcode == Opcode::GetElementPtr
+    }
+
+    pub(crate) fn is_control_point(&self, aot_mod: &Module) -> bool {
+        if self.opcode == Opcode::Call {
+            let op = &self.operands[0];
+            match op {
+                Operand::Function(fop) => {
+                    return aot_mod.funcs[fop.func_idx].name == "__ykrt_control_point";
```

Comment:
Probably best as a constant somewhere as we're bound to need it more than once.

Reply:
Fixed in a4a13b1bd36cc180c6757a28b0099726db519f10